Dockerfile

```diff
@@ -1,29 +1,55 @@
-#
-# Dockerfile for ytdl-webserver
-#
-FROM ubuntu:16.04
+FROM alpine:3.8 as base
 
 WORKDIR /home/app
+EXPOSE 3000
+CMD [ "npm", "start" ]
 
-RUN apt-get update \
-    && apt-get install -y curl ffmpeg \
-    && curl -sL https://deb.nodesource.com/setup_6.x | bash - \
-    && apt-get install -y nodejs \
-    && npm install -g npm@latest \
-    && rm -rf /var/lib/apt/lists/*
+RUN apk add --update --no-cache \
+        python3 \
+        ffmpeg \
+        nodejs \
+        nodejs-npm \
+        ca-certificates && \
+    find / -type d -name __pycache__ -exec rm -r {} +   && \
+    rm -r /usr/lib/python*/ensurepip                    && \
+    rm -r /usr/lib/python*/lib2to3                      && \
+    rm -r /usr/lib/python*/turtledemo                   && \
+    rm /usr/lib/python*/turtle.py                       && \
+    rm /usr/lib/python*/webbrowser.py                   && \
+    rm /usr/lib/python*/doctest.py                      && \
+    rm /usr/lib/python*/pydoc.py                        && \
+    rm -rf /root/.cache /var/cache /usr/share/terminfo  && \
+    ln /usr/bin/python3.6 /usr/bin/python               && \
+    mkdir -p public/temp
 
-# This is on a separate line because youtube-dl needs to be frequently updated
-RUN apt-get update \
-    && apt-get install -y youtube-dl \
-    && rm -rf /var/lib/apt/lists/*
+# install the latest npm and clear the cache
+RUN npm install -g npm@latest && \
```

Comment:
might as well just merge this with above RUN directive to save on layers, right?